Common/Transforms/itkRecursiveBSplineTransformImplementation.h

```diff
@@ -533,22 +537,13 @@ class RecursiveBSplineTransformImplementation<OutputDimension, 0, SplineOrder, T
      */
     if (OutputDimension == 3)
     {
-      jsh_tmp[0] = jsh[9];
-      jsh_tmp[1] = jsh[8];
-      jsh_tmp[2] = jsh[7];
-      jsh_tmp[3] = jsh_tmp[1];
-      jsh_tmp[4] = jsh[5];
-      jsh_tmp[5] = jsh[4];
-      jsh_tmp[6] = jsh_tmp[2];
-      jsh_tmp[7] = jsh_tmp[5];
-      jsh_tmp[8] = jsh[2];
+      const double tmp[] = { jsh[9], jsh[8], jsh[7], jsh[8], jsh[5], jsh[4], jsh[7], jsh[4], jsh[2] };
```

Comment:
are these numbers correct? in the old code it reads 9 8 7 then 1 ...

Reply:
@mstaring Thanks for asking, Marius! Honestly I did double check  😃 The old code did `jsh_tmp[3] = jsh_tmp[1]`, knowing that  jsh_tmp[1] == jsh[8]. So the old code did 9 8 7 then _indirectly_ 8, via `jsh_tmp[1]`!

So the new code _might_ even be faster than the old one, because it does 9 8 7 8 _directly_!  (I say _might_ because I did not do any benchmarking.)

Reply:
Ah yes, i see now. Thank you!